README.md

```diff
@@ -106,6 +106,41 @@ cd Scripts
 pipenv run python helmet.py
 ```
 
+
+### Mock assignment run in Helmet UI
+
+Download [Helmet-UI executable](https://github.com/HSLdevcom/helmet-ui/releases/). 
```

Comment:
Refer to this manual: https://hsldevcom.github.io/helmet-docs/kaytto-ohje.html#helmet-asennus